From 373809e180c9f07a09ee1eaa80df19075855c95d Mon Sep 17 00:00:00 2001
From: rolandlo <roland_loetscher@hotmail.com>
Date: Mon, 19 Jul 2021 23:54:28 +0200
Subject: [PATCH 1/2] Update appdata screenshots

In order to fit resolution constraints for appstream-util validate
---
 desktop/com.github.xournalpp.xournalpp.appdata.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/desktop/com.github.xournalpp.xournalpp.appdata.xml b/desktop/com.github.xournalpp.xournalpp.appdata.xml
index 06ee132d..56d4a807 100644
--- a/desktop/com.github.xournalpp.xournalpp.appdata.xml
+++ b/desktop/com.github.xournalpp.xournalpp.appdata.xml
@@ -52,13 +52,13 @@
   <url type="homepage">https://xournalpp.github.io/</url>
   <screenshots>
     <screenshot type="default">
-      <image>https://user-images.githubusercontent.com/40485433/122611342-70fe3a80-d081-11eb-916a-92e570a1a416.png</image>
+      <image>https://raw.githubusercontent.com/xournalpp/xournalpp-examples/main/appdata/screenshots/note_taking_1600x900.png</image>
     </screenshot>
     <screenshot>
-      <image>https://user-images.githubusercontent.com/40485433/122599842-42776400-d06f-11eb-82a2-c006dcd0c36a.png</image>
+      <image>https://raw.githubusercontent.com/xournalpp/xournalpp-examples/main/appdata/screenshots/pdf_annotation_1600x900.png</image>
     </screenshot>
     <screenshot>
-      <image>https://user-images.githubusercontent.com/40485433/122599858-47d4ae80-d06f-11eb-9dc4-1e546c78b632.png</image>
+      <image>https://raw.githubusercontent.com/xournalpp/xournalpp-examples/main/appdata/screenshots/LaTeX_with_customized_preamble_1600x900.png</image>
     </screenshot>
   </screenshots>
   <launchable type="desktop-id">com.github.xournalpp.xournalpp.desktop</launchable>
-- 
2.30.2

